packages/web/pages/_app.tsx

```diff
@@ -28,7 +29,7 @@ dayjs.extend(utc);
 enableStaticRendering(typeof window === "undefined");
 
 function MyApp({ Component, pageProps }: AppProps) {
-  const menus = [
+  let menus: MainLayoutMenu[] = [
```

Comment:
It seems that It don't need to change declare type.
```suggestion
  const menus: MainLayoutMenu[] = [
```